src/functions/math.rs

```diff
@@ -242,6 +268,12 @@ fn number(v: Rational, unit: Unit) -> Value {
     Value::Numeric(Number::from(v), unit, true)
 }
 
+/// convert f64 in radians (used by rust) to numeric Value in degrees
+/// (used by sass).
+fn deg_value(rad: f64) -> Value {
+    float_value(rad * 180. / std::f64::consts::PI, Unit::Deg)
```

Comment:
There is https://doc.rust-lang.org/std/primitive.f64.html#method.to_degrees

Reply:
Ah, thanks!